pages/instructorIssues/instructorIssues.js

```diff
@@ -1,17 +1,91 @@
-var ERR = require('async-stacktrace');
-var _ = require('lodash');
-var express = require('express');
-var router = express.Router();
+const ERR = require('async-stacktrace');
+const _ = require('lodash');
+const moment = require('moment');
+const express = require('express');
+const router = express.Router();
+const SearchString = require('search-string');
 
-var error = require('../../lib/error');
-var paginate = require('../../lib/paginate');
-var sqldb = require('../../lib/sqldb');
-var sqlLoader = require('../../lib/sql-loader');
+const error = require('../../lib/error');
+const paginate = require('../../lib/paginate');
+const sqldb = require('../../lib/sqldb');
+const sqlLoader = require('../../lib/sql-loader');
 
-var sql = sqlLoader.loadSqlEquiv(__filename);
+const sql = sqlLoader.loadSqlEquiv(__filename);
 
 const pageSize = 100;
 
+const commonQueries = {
+    allOpenQuery: 'is:open',
+    allClosedQuery: 'is:closed',
+    allManuallyReportedQuery: 'is:manually-reported',
+};
+
```

Comment:
What do you think about adding github-style dropdowns to filter by QID/UID? That is, like the github dropdowns to filter by Author, Labels, etc. These have a text entry box where you can type the QID (maybe with autocomplete using `<datalist>`?), or pick it from a list. The lists might be too long for us, but the text entry box would be more discoverable than reading the help message about typing `qid:QID` (which is also a good feature).

Reply:
Yeah, was already thinking about doing something like that as a future enhancement. Having a UI in additional to typing in filters is a good idea for usability.